core/core_bind.cpp

```diff
@@ -1897,6 +1897,14 @@ bool Engine::is_editor_hint() const {
 	return ::Engine::get_singleton()->is_editor_hint();
 }
 
+void Engine::set_embedded_in_editor(bool p_enabled) {
```

Comment:
Why does this exist if it's not exposed?

Reply:
It was not used, I just copied the `set_editor_hint` and `is_editor_hint`. I removed it.